release-notes-3.7.0.md

```diff
@@ -1,8 +1,15 @@
-!not-ready-for-release!
 
 #### Version Number
 ${version-number}
 
 #### New Features
```

Comment:
Just a couple of things Xavier:

- Are all these new features, ie. none of these are bug fixes?

- We normally provide a little more information in each - it's normally not enough just to copy the commit message into the release notes (sometimes it's ok if it's good enough). See here for examples of the sort of information to provide: https://github.com/WorkerFramework/worker-framework/releases/tag/v1.6.0

Reply:
Also, we only include important fixes, so I'd argue that:

 **SCMOD-11683**: Decreased log level to Info
 
 should not be included in release notes, as I believe it just affects the integration tests.
 
 https://portal.digitalsafe.net/browse/SCMOD-11987
 
"New Features" section
List important new features and reference appropriate Jiras
Do not list trivial features or try to justify every commit
Do not list pet features (unless they are important)
"Bug Fixes" section
List important fixes, esp. issues called out in the "Known Issues" section of previous releases.
Do not list trivial fixes
"Known Issues" section
List important issues that we will need to fix in an upcoming release
Do not list trivial issues